### PR TITLE
LPS-72504 Creates the icon soy taglib

### PR DESCRIPTION
**Pull request by fernandosouza**
Bringing the comment from the original pull request:

> I would like to point that this version drops the label attribute of the JSP version for while. From what I understood, the JSP taglib uses the label to write inside the <span> element or to set the attribute title of the <svg> element. In the first case, it looks like another component the has an icon, for example, an <button> or <a>. In the second case, it tries to implement accessibility, but I'm not sure if this is the right approach. I'm doing a research about accessibility for icons.

cc. @carloslancha, @jbalsas

### CODE REVIEW

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.soy [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * @param changeableDefaultLanguage
  * @param defaultLocale
  * @param? elementClasses
- * @param? id
+ * @param id
  * @param locales
```
Comment (brunobasto) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.soy]: @fernandosouza Can you define the parameters using the new syntax? See https://github.com/liferay/com-liferay-osb-faro-build-private/wiki/Metal-Soy-Guidelines#declaring-params

File: modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java [hunk 2]
```diff
@@ -85,12 +84,18 @@ public int doStartTag() {
 		return EVAL_BODY_INCLUDE;
 	}
 
+	/**
+	* @deprecated As of 1.0.0, replaced by {@link #getId()}
```
Comment (brunobasto) [inline modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java]: I believe the correct version is 1.1.0 now

File: modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java [hunk 3]
```diff
@@ -118,14 +123,22 @@ public void putValue(String key, Object value) {
 		context.put(key, value);
 	}
 
+	/**
+	* @deprecated As of 1.0.0, replaced by {@link #setId(String)}
```
Comment (brunobasto) [inline modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java]: I believe the correct version is 1.1.0 now

**Comment by brunobasto** [comment]
@jbalsas is this good to merge?

Thx

File: modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java (hunk 2 above)
Comment (fernandosouza) [inline modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java]: Ok. I thought I should mention the previous version. I'm gonna fix it.

File: modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java (hunk 3 above)
Comment (fernandosouza) [inline modules/apps/foundation/frontend-taglib/frontend-taglib-soy/src/main/java/com/liferay/frontend/taglib/soy/servlet/taglib/TemplateRendererTag.java]: I'm gonna fix it.

File: modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.soy (hunk 1 above)
Comment (fernandosouza) [inline modules/apps/foundation/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/TranslationManager.soy]: Sure!

**Comment by fernandosouza** [comment]
Hey, guys. I found issues in this PR. I'm going to resend.

**Comment by fernandosouza** [comment]
@brunobasto, branch updated. Can you please take a look again?

**Comment by fernandosouza** [comment]
ci:retest

**Pull request closed without merging**